doc/source/installation/yum_repo.rst

```diff
@@ -18,6 +18,36 @@ Supported Releases:
 
 The *CentOS* repositories should work well with *Red Hat Enterprise Linux* too, provided that :program:`yum` is installed on the server.
 
+.. important::
+
+   *CentOS* 6 offers an outdated version of the ``curl`` library required by
+    the keyring Vault plugin of |Percona Server|. The version of the ``curl`` library in *CentOS* 6, which
+    depends on the ``nss`` library, is known to create memory corruption
+    issues. This bug is `registered in Red Hat Bugzilla
+    <https://bugzilla.redhat.com/show_bug.cgi?id=1057388>`_. Its current status
+    is `CLOSED WONTFIX`.
+
+    If you intend to use the keyring Vault plugin of |Percona Server|
+    make sure that you use the latest version of the ``curl`` library.
+    We recommend that you `build it from source
+    <https://curl.haxx.se/docs/install.html>`_ configuring with ssl:
+
+    .. code-block:: bash
+
+       $ ./configuration --with-ssl --prefix=<INSTALATION DIRECTORY>
+
+    As soon as you install ``curl``, make sure that |Percona Server| will use
+    this version, setting the environment variable ``LD_LIBRARY_PATH``:
+
+    .. code-block:: bash
+
+       $ export LD_LIBRARY_PATH=<INSTALATION DIRECTORY>/lib:$LD_LIBRARY_PATH
```

Comment:
it seems my comment in PS-5479 has been overlooked. Please have a look. It is about getting rid of describing LD_LIBRARY_PATH.